scss/bitstyles/tools/_color.scss

```diff
@@ -15,10 +15,10 @@
 // @param {list} $percentages  A list of decmial values representing the percentages at which the mix-color will be mixed into the base-color
 
 @function generate-palette($base-color, $mix-color, $percentages) {
-  $color-map: ();
+  $color-map: (); // stylelint-disable-line scss/dollar-variable-default
```

Comment:
Hm, a bit weird that the linter does not get the function context here 🤔 

Does the fix from [this issue](https://github.com/kristerkari/stylelint-scss/issues/76) make a difference?

Reply:
It did! I updated the linting rules, thanks 🎉